searx/settings.yml

```diff
@@ -273,6 +273,15 @@ engines:
     categories : images
     shortcut : cce
 
+  - name : core.ac.uk
+    engine : core
+    categories : science
+    shortcut : cor
+    # get your API key from: https://core.ac.uk/api-keys/register/
+    # api_key : "xxxxxxxx"
+    # set api_key and comment out disabled ..
+    disabled: True
+
```

Comment:
It is the whole engine definition that should be comment out : without the api_key nothing works AFIK.